tests/async_tests/test_interfaces.py

```diff
@@ -204,6 +205,46 @@ async def test_http_proxy(
         assert reason == b"OK"
 
 
+@pytest.mark.parametrize("family", [socket.AF_INET])
+@pytest.mark.asyncio
+# This doesn't run with trio, since trio doesn't support family.
+async def test_http_request_family(family: int,) -> None:
+    async with httpcore.AsyncConnectionPool(family=family) as http:
+        method = b"GET"
+        url = (b"http", b"example.org", 80, b"/")
+        headers = [(b"host", b"example.org")]
+        http_version, status_code, reason, headers, stream = await http.request(
+            method, url, headers
+        )
+        body = await read_body(stream)
+
+        assert http_version == b"HTTP/1.1"
+        assert status_code == 200
+        assert reason == b"OK"
+        assert len(http._connections[url[:3]]) == 1  # type: ignore
+
+
+@pytest.mark.parametrize("local_addr", ["0.0.0.0"])
+@pytest.mark.asyncio
+# This doesn't run with trio, since trio doesn't support local_addr.
+async def test_http_request_local_addr(local_addr: str) -> None:
+    family = socket.AF_INET6 if ':' in local_addr else socket.AF_INET
```

Comment:
This seems leftover from when there was more than one value in the parametrization, might be worth removing it altogether?

Reply:
The test really should be trying more than one case, but when I tried testing IPv6, it failed due to the test system not supporting IPv6.  I'd rather leave this to make it more clear that it should be parametrized, but if it's important, it can be removed.